src/test/java/org/benetech/mathshare/converters/UrlCodeConverterUtilsTest.java

```diff
@@ -0,0 +1,119 @@
+package org.benetech.mathshare.converters;
+
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+public class UrlCodeConverterUtilsTest {
+
+    @Test(expected = IllegalArgumentException.class)
+    public void fromUrlCodeMethodShouldThrowIllegalArgumentExceptionForNotBase32AndNotXCharacters() {
```

Comment:
I know it is a convention but shouldn't we shorten such long test method names? e.g. remove part 'fromUrlCodeMethod'. IMO the current readability is poor.

Reply:
In my opinion, it's even more readable which method is tested, as the converter has more than one method.

Reply:
ok